src/libraries/Microsoft.Extensions.DependencyInjection.Abstractions/src/Extensions/ServiceCollectionDescriptorExtensions.Keyed.cs

```diff
@@ -0,0 +1,416 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Microsoft.Extensions.DependencyInjection.Extensions
+{
+    /// <summary>
+    /// Extension methods for adding and removing services to an <see cref="IServiceCollection" />.
```

Comment:
This is a partial class and the other class also has the same doc which I don't think we need in both places.

Also nit: extra whitespace before `/>`